jwst/extract_1d/extract_1d_step.py

```diff
@@ -196,4 +196,7 @@ def process(self, input):
 
         input_model.close()
 
+        if len(result) and result[0].meta.exposure.type == 'NIS_SOSS':
```

Comment:
What about other modes, such as NRC_WFSS and NIS_WFSS, that might need to be handled similarly to SOSS? Or do they use a different type of model/container that doesn't apply here?